db/make_db.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
This is enough to init it:
 - https://github.com/pierreozoux/mattermost-docker/blob/cdf8330e9fb0b4d4197fe6a6436574c9d7d4c060/db/Dockerfile#L15-L16
 - https://hub.docker.com/_/postgres/